src/__tests__/helpers/splitToRegisters.ts

```diff
@@ -0,0 +1,24 @@
+import * as BN from 'bn.js'
+
+const REGISTERS = 4n
+
+function addHexPrefix(str: string) {
+  return `0x${str}`
+}
+
+export default function (value?: BN | string) {
+  const registers = [] as bigint[]
+
+  if (!value) {
```

Comment:
Will this be one line of we remove curlies? If yes, let's remove them

Reply:
fixed